### PR TITLE
Completed html-forms exercise. - Gino Hong

### PR DESCRIPTION
**Pull request by gino-hong**
html-forms

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      </div>
+      <input type="radio" name="language" value="html"> HTML
+      <input type="radio" name="language" value="css"> CSS
+      <input type="radio" name="language" value="javascript"> JavaScript
+      <div>
+        <label for="signup">Would you like to sign up for more information?</label>
+      </div>
+      <input type="checkbox" name="signup" value="signup"> Sign me up!
+      <div>
+        <label for="comments">Comments</label>
```
Comment (BrettAlbright) [inline html-forms/index.html:39]: the input needs a matching `id` attribute for the labels to work correctly

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        <label for="coding">Why is coding cool?</label>
+      </div>
+      <select name="coding" id="coding">
+        <option>Select an option</option>
+        <option value="interesting" >Because it's interesting</option>
+        <option value="fun">Because it's fun</option>
+        <option value="powerful">Because it's powerful</option>
+      </select>
+      <div>
+        <label for="language">Favorite Language (choose one)</label>
```
Comment (BrettAlbright) [inline html-forms/index.html]: you will need 3 labels here, and they can be wrapped like this: 
```html
      <label>
        <input type="radio" name="favoriteLanguage" value="JavaScript">
        JavaScript
      </label>
```

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
a couple small updates!

**Comment by gino-hong** [comment]
html-forms

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**